src/main/java/dev/warrant/model/QueryResultSet.java

```diff
@@ -3,22 +3,40 @@
 public class QueryResultSet {
 
     private QueryResult[] results;
-    private String lastId;
+    private String prevCursor;
+    private String nextCursor;
 
     public QueryResultSet() {
         // For json serialization
     }
 
-    public QueryResultSet(QueryResult[] results, String lastId) {
+    public QueryResultSet(QueryResult[] results) {
```

Comment:
I think we can get rid of this class and instead use `ListResult<QueryResult>` once we update `ListResult` to take any type (and not just those that implement `WarrantObject`).

Reply:
We have `QueryResultSet` because this gets passed into `makeGetRequest` to map the response into the specified class, which we wouldn't be able to do with `ListResult<QueryResult>`